diff --git a/src/apps/aboutsystem/AboutSystem.cpp b/src/apps/aboutsystem/AboutSystem.cpp
index 2b59c4e1..16b07035 100644
--- a/src/apps/aboutsystem/AboutSystem.cpp
+++ b/src/apps/aboutsystem/AboutSystem.cpp
@@ -1770,7 +1770,7 @@ AboutView::_CreateCreditsView()
 		B_TRANSLATE("Contains software from the FreeBSD Project, "
 		"released under the BSD license:\n"
 		"ftpd, ping, telnet, telnetd, traceroute\n"
-		COPYRIGHT_STRING "1994-2008 The FreeBSD Project. "
+		COPYRIGHT_STRING "1994-2024 The FreeBSD Project. "
 		"All rights reserved."),
 		StringVector(kBSDTwoClause, kBSDThreeClause, kBSDFourClause,
 			NULL),
@@ -1790,6 +1790,17 @@ AboutView::_CreateCreditsView()
 		StringVector(),
 		"https://www.netbsd.org");
 
+	// OpenBSD copyrights
+	AddCopyrightEntry("The OpenBSD Project",
+		B_TRANSLATE("Contains software from the OpenBSD Project, "
+		"released under the ISC license:\n"
+		"iaxwifi200 (Intel AX200)\n"
+		COPYRIGHT_STRING "2014-2020 Stefan Sperling, et al. "
+		"All rights reserved."),
+		StringVector("ISC", NULL),
+		StringVector(),
+		"https://www.openbsd.org");
+
 	// FFmpeg copyrights
 	_AddPackageCredit(PackageCredit("FFmpeg")
 		.SetCopyright(B_TRANSLATE(COPYRIGHT_STRING "2000-2019 Fabrice "
@@ -1806,9 +1817,9 @@ AboutView::_CreateCreditsView()
 
 	// FreeType copyrights
 	_AddPackageCredit(PackageCredit("FreeType2")
-		.SetCopyrights(B_TRANSLATE(COPYRIGHT_STRING "1996-2002, 2006 "
+		.SetCopyrights(B_TRANSLATE(COPYRIGHT_STRING "1996-2026 "
 			"David Turner, Robert Wilhelm and Werner Lemberg."),
-			COPYRIGHT_STRING "2014 The FreeType Project. "
+			COPYRIGHT_STRING "2006-2026 The FreeType Project. "
 			"All rights reserved.",
 			NULL)
 		.SetLicense("FreeType")
@@ -1816,7 +1827,7 @@ AboutView::_CreateCreditsView()
 
 	// Mesa3D (http://www.mesa3d.org) copyrights
 	_AddPackageCredit(PackageCredit("Mesa")
-		.SetCopyright(B_TRANSLATE(COPYRIGHT_STRING "1999-2006 Brian Paul. "
+		.SetCopyright(B_TRANSLATE(COPYRIGHT_STRING "1999-2026 Brian Paul. "
 			"Mesa3D Project. All rights reserved."))
 		.SetLicense("MIT")
 		.SetURL("https://www.mesa3d.org"));
@@ -1869,7 +1880,7 @@ AboutView::_CreateCreditsView()
 
 	// zlib copyrights
 	_AddPackageCredit(PackageCredit("zlib")
-		.SetCopyright(B_TRANSLATE(COPYRIGHT_STRING "1995-2004 Jean-loup "
+		.SetCopyright(B_TRANSLATE(COPYRIGHT_STRING "1995-2026 Jean-loup "
 			"Gailly and Mark Adler."))
 		.SetLicense("Zlib")
 		.SetURL("https://www.zlib.net"));
@@ -1890,19 +1901,19 @@ AboutView::_CreateCreditsView()
 
 	// acpica copyrights
 	_AddPackageCredit(PackageCredit("ACPI Component Architecture (ACPICA)")
-		.SetCopyright(COPYRIGHT_STRING "1999-2018 Intel Corp.")
+		.SetCopyright(COPYRIGHT_STRING "1999-2026 Intel Corp.")
 		.SetLicense("Intel (ACPICA)")
 		.SetURL("https://www.intel.com/content/www/us/en/developer/topic-technology/open/acpica/overview.html"));
 
 	// libpng copyrights
 	_AddPackageCredit(PackageCredit("libpng")
-		.SetCopyright(COPYRIGHT_STRING "1995-2017 libpng authors")
+		.SetCopyright(COPYRIGHT_STRING "1995-2025 libpng authors")
 		.SetLicense("LibPNG")
 		.SetURL("http://www.libpng.org"));
 
 	// libjpeg copyrights
 	_AddPackageCredit(PackageCredit("libjpeg")
-		.SetCopyright(B_TRANSLATE(COPYRIGHT_STRING "1994-2009, Thomas G. "
+		.SetCopyright(B_TRANSLATE(COPYRIGHT_STRING "1991-2019, Thomas G. "
 			"Lane, Guido Vollbeding. This software is based in part on the "
 			"work of the Independent JPEG Group."))
 		.SetLicense("LibJPEG")
@@ -1930,7 +1941,7 @@ AboutView::_CreateCreditsView()
 	// Xiph.org Foundation copyrights
 	_AddPackageCredit(PackageCredit("Xiph.org Foundation")
 		.SetCopyrights("libvorbis, libogg, libtheora, libspeex",
-			B_TRANSLATE(COPYRIGHT_STRING "1994-2008 Xiph.Org. "
+			B_TRANSLATE(COPYRIGHT_STRING "1994-2018 Xiph.Org. "
 			"All rights reserved."), NULL)
 		.SetLicense(kBSDThreeClause)
 		.SetURL("https://www.xiph.org"));
@@ -2010,14 +2021,14 @@ AboutView::_CreateCreditsView()
 	// Gutenprint
 	_AddPackageCredit(PackageCredit("Gutenprint")
 		.SetCopyright(B_TRANSLATE(COPYRIGHT_STRING
-			"1999-2010 by the authors of Gutenprint. All rights reserved."))
+			"1999-2020 by the authors of Gutenprint. All rights reserved."))
 		.SetLicense(kGPLv2)
 		.SetURL("https://gimp-print.sourceforge.io"));
 
 	// libwebp
 	_AddPackageCredit(PackageCredit("libwebp")
 		.SetCopyright(B_TRANSLATE(COPYRIGHT_STRING
-			"2010-2011 Google Inc. All rights reserved."))
+			"2011-2025 Google Inc. All rights reserved."))
 		.SetLicense(kBSDThreeClause)
 		.SetURL("https://www.webmproject.org/code/#webp-repositories"));
 
